source/blog/2018-07-16-ember-3-3-released.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Ember 3.3 Released
+author: Melanie Sumner, Brendan McLoughlin
+tags: Releases, 2018, 3, 3.3
+responsive: true
+---
+
+Today the Ember project is releasing version 3.3.0 of Ember.js, Ember Data, and Ember CLI. We know, 3.2 seems like it just came out yesterday! But we wanted to (stay? get back?) on track. So 3.3.0 is here! 
```

Comment:
Not sure if it is worth mentioning, but the reasoning for continuing 3.3 today was that Ember had already completed a beta cycle, so it's really ember-data & ember-cli catching up with Ember.